proto/regen/ecocredit/v1/tx.proto

```diff
@@ -111,8 +112,23 @@ service Msg {
   // project with a matching reference id already exists within the scope of the
   // credit class, otherwise a new project will be created.
   rpc BridgeReceive(MsgBridgeReceive) returns (MsgBridgeReceiveResponse);
+
+  // AddCreditType is a governance method that allows the addition of new credit types to the network.
+  rpc AddCreditType(MsgAddCreditType) returns (MsgAddCreditTypeResponse);
 }
 
+// MsgAddCreditType is the Msg/AddCreditType request type.
+message MsgAddCreditType {
+  // authority is the address of the governance account.
+  string authority = 1;
+
+  // credit_type defines a credit type to add to the credit types parameter. 
+  repeated CreditType credit_type = 2;
+}
```

Comment:
I think we should only allow one credit type at a time. This will not be frequent and make things a bit simpler.